### DISCUSSION TITLE
Determining what processes are using Memory (raven)

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1402] [Q&A]
It would appear raven is being taxed on memory 

```
sr320@raven:/home/shared$ free
              total        used        free      shared  buff/cache   available
Mem:      259449556   257800120     1441456          36      207980     1460260
Swap:     104857596    91414472    13443124
```

though `htop` is not giving any clues...

<img width="409" alt="nb-2022__RStudio_Server_2022-02-16_14-48-36" src="https://user-images.githubusercontent.com/4442039/154370735-2ea1cb13-8e6d-41be-8fd5-79b0f16bd02e.png">

any ideas on determining the culprit?

**Comment by kubu4** [comment]
> any ideas on determining the culprit?

Looks like bedtools (the top of the list, sorted by MEM%) is utilizing most of the available memory.

**Reply by sr320** [reply]
Yep! Right there in line 1 😬
